landoapi/transplants.py

```diff
@@ -867,6 +867,18 @@ def blocker_prevent_symlinks(
         )
 
 
+def blocker_try_task_config(
+    revision: dict, diff: dict, stack_state: StackAssessmentState
+) -> Optional[str]:
+    """Block revisions which contain the `try_task_config.json` file."""
+    diff_id = PhabricatorClient.expect(diff, "id")
+    parsed_diff = stack_state.parsed_diffs[diff_id]
+
+    for parsed in parsed_diff:
+        if parsed["filename"] == "try_task_config.json":
+            return "Revision introduces the `try_task_config.json` file into a non-try repo."
```

Comment:
I don't think this error string is accurate any more, given that we are not checking the repo anywhere.

```suggestion
            return "Revision introduces the `try_task_config.json` file."
```